ParkingSlots.Tests/Controller/ParkingSlotControllerTests.cs

```diff
@@ -66,7 +67,100 @@ public void GivenParkingZoneId_WhenIndexIsCalled_ThenReturnsParkingSlotsVM()
         Assert.Equal(JsonSerializer.Serialize(model), JsonSerializer.Serialize(expectedVMs));
         Assert.NotNull(index);
         Assert.NotNull(model);
-        _slotService.Verify(_parkingSlot => _parkingSlot.GetSlotsByZoneId(_Id), Times.Once);
+        _slotService.Verify(_parkingSlot => _parkingSlot.GetSlotsByZoneId(_id), Times.Once);
+    }
+    #endregion
+
+    #region Create
+    [Fact]
+    public void GivenParkingZoneId_WhenCreateIsCalled_ThenReturnsCreateViewModel()
+    {
+        //Arrange
+        var createVM = new CreateViewModel() { ParkingZoneId = _id };
+
+        //Act
+        var create = _controller.Create(_id);
+        var model = ((ViewResult)create).Model;
```

Comment:
In Act part why are you taking model
It should be in Assert part
var model = Assert.IsType<ViewResult>(create).Model;